python_operator/template.py

```diff
@@ -74,6 +74,7 @@ def create_charm(self, config, output_dir):
                 self._template_file(config, path.join(root, outfile))
 
         self._add_submodule(output_dir)
+        self._git_commit(output_dir)
```

Comment:
We should specify a charm commit message like "Created charm using `charm create` based on the python operator template"

Reply:
Pushing an update that calls out the charm-template creation and the operator template as the source.